admin.php

```diff
@@ -26,6 +26,11 @@ function forAdminOnly() {
      * Handle tag actions
      */
     function handle() {
+
+        if (!empty($_REQUEST['cmd']['clean'])) {
```

Comment:
I think this should check the securityToken to avoid CSRF attacks triggering the deletions of the tags

Reply:
fixed in a88577ddfa97b6eb7296192d1d65f0dd6ccc58e7